pydantic_ai_slim/pydantic_ai/__init__.py

```diff
@@ -95,6 +95,18 @@
     ModelProfile,
     ModelProfileSpec,
 )
+from .prompt_config import (
+    DEFAULT_FINAL_RESULT_PROCESSED,
```

Comment:
I don't love exporting so many constants here, but see your point below about having to cast fields to `str` when using `DEFAULT_PROMPT_CONFIG`. Still leaving this comment here as a sign of "let's get back to this" :)